### DISCUSSION TITLE
Three DS18B20 sensors at one GPIO, how to get the values by script

### DISCUSSION THREAD

**Discussion by Fast63947** [arendst/Tasmota#21237] [Q&A] [answered]
Hello,

I have connected tree DS18B20 at one GPIO of a ESP8266 and the values will be shown in the web interface. Now I want to get the values in the script. How to address the sensors int the script. I I connect only one sensor I can get the value by using 'tempVar=DS18B20#Temperature' but using three sensors I don't know, how to get the values. Who can help?

**Comment by sfromis** [comment, marked as answer]
You need to use the names for each sensor, as visible in the sensor payloads you get in the console, and make sure to have `SetOption64` set for using underscore as a delimiter, giving names like `DS18B20_2#Temperature`.

**Comment by Fast63947** [comment]
You are great! I did not know about the SetOption64 for using the underline. Now it works perfekt. Thank you very much for the fast help.
